src/pat/inject/inject.js

```diff
@@ -206,6 +207,14 @@ define([
                     cfg.$target.removeClass('is-dirty');
                 }
 
+                var $form = cfg.$target.parents('form')
+                if ($form.size() !== 0 && cfg.$target.data('initial-value') === undefined) {
+                    cfg.$target.data('initial-value', cfg.$target.html());
+                    $form.on('reset', function() {
+                        cfg.$target.html(cfg.$target.data('initial-value'))
+                    })
+                }
+
```

Comment:
will add the comment. 

just to explain it here. pat-inject is used to populate target in some ``form`` and when ``Cancel`` button is presed (this triggers ``reset`` event on the form) you would expect to populate with initial placeholder 

example for this is attachements at dashboard in ploneintranet